CVInHtml/cv.html

```diff
@@ -0,0 +1,118 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>CV exercise</title>
+</head>
+<body>
+    <header>
+        <h1>Resume</h1>
+        <nav>
+            <ul>
+                <li><a href="#about">About</a></li>
+                <li><a href="#skills">Skills</a></li>
+                <li><a href="#experience">Experience</a></li>
+                <li><a href="#contact">Contact</a></li>
+            </ul>
+        </nav>
+    </header>
+    <main>
+        <section id="about">
+            <h2>About</h2>
+            <p>
+                Jonathan Doe
+            </p>
+            <p>
+                Web Designer
+            </p>
+            <img src="images.png" alt="profile picture">
```

Comment:
wrong image source

Reply:
Image source changed from images.png to images/image.webp